en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/security/api-authentication/configuring-an-external-key-manager.md

```diff
@@ -13,7 +22,39 @@ Please refer [Multiple Key Manager Support in WSO2 API Manager]({{base_path}}/ad
 !!! note
     Please note that Choreo Connect only supports self-validation of JWT tokens from key managers. (JWT tokens will be validated against the issuer data.)
 
-!!! note
+!!! important
     Token services which are added from the `config.toml` file under `enforcer.security.tokenService` configuration will be overridden from the retrieved key manager configurations from the API Manager if the issuers are identical. Furthermore, if the corresponding key manager is removed from the API Manager admin portal, the token service added from the configuration will be used.
 
-
+## Via the Choreo Connect Config File
+
+When Choreo Connect does not depend on a control plane, the external Key Managers, Token Services or JWT issuers can be configured in [config.toml]({{base_path}}/deploy-and-publish/deploy-on-gateway/choreo-connect/configurations/configuration-overview/#configurations-overview) as given below.
+
```

Comment:
Isn't it better to provide a guide on how to configure the following tokenservice. feels this section is missing info

Reply:
Can you give an example?

Reply:
we have just given the config example without explaining much on how to identify/provide values for the config, haven't we?

Reply:
Got the point. I also have given a generic link above. Let me update it together with some updates to the config descriptions.